packages/suite/src/reducers/suite/metadataReducer.ts

```diff
@@ -91,6 +91,11 @@ const metadataReducer = (state = initialState, action: Action): MetadataState =>
                     delete draft.error?.[action.payload.deviceState];
                 }
                 break;
+            case '@suite/device/forgetDevice':
```

Comment:
🙏 deviceActions.forgetDevice.type instead of string

Reply:
7b616c569f55127e9866d88088d2bbc5fe2fd2f5